tests/components/tuya/test_light.py

```diff
@@ -42,11 +43,56 @@ async def test_platform_setup_and_discovery(
     "mock_device_code",
     ["dj_mki13ie507rlry4r"],
 )
+@pytest.mark.parametrize(
+    ("turn_on_input", "expected_commands"),
+    [
+        (
+            {"white": True},
```

Comment:
Nit-pick: align all inputs
```suggestion
            {
                "white": 150,
            },
```

Reply:
yes it should be. i will just commit your suggestion to make it work